data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+				goto teeneepquestions
+			label listen
+			action
+				set "Hai Reveal: will the elders listen?"
+			`	She sighs. "I wish to believe so, but part of me I can no longer ignore does not think it actually likely. Still, it is essential that I try with everything I have to make them do so. Otherwise there will be change, and we can be an unpredictable people in the face of that."`
+				goto teeneepquestions
+			label war
+			action
+				set "Hai Reveal: will there be war?"
+			`	She frowns and gently shakes her head. "No, or at least, not yet. The willingness of the Unfettered to threaten war is essential to all of the possible paths from here. Even if the Elders refuse to take action though, there are more steps between this meeting and that outcome. The options are not yet exhausted, though they are rapidly dwindling. With luck though, Aimet's influence might tip the situation in our favor."`
+				goto teeneepquestions
+			label options
+			`	"Ah, well those are difficult to explain. Since the options worth using will depend on exactly what happens in the meeting. One thing Sayari was very right to remind me off before is that there are normal channels and processes to follow, and those must be followed. We're an old people <last>, and not always a very patient people. We've forged a system of government with systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together."`
+			`	Her mouth quirks in a sly little smile, "Aimet has always been the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years." She frowns. "It's actually very odd that he wasn't chosen for the Triad last time we were here."`
+				goto return
+			label aimet
+			`	"He is the," she pauses, "I don't think it translates quite right. You have a phenomenon in your language where you amalgamate words sometimes, like an engineer who works specifically on solutions to problems may be colloquially referred to as a 'solutioneer'. We have a similar thing with our words for political systems and navigator. It's a title he has earned."`
+			`	She spreads her arms, 'palms' up, in a human gesture indicating a loss for words, "Our system of government has systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together. Aimet is the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years."`
```

Comment:
```suggestion
			`	She spreads her arms, 'palms' up, in a human gesture indicating a loss for words. "Our system of government has systems and processes to allow almost any internal situation to be handled without resorting to base hysteria, and to actively punish and isolate those who try and subvert those processes. It is a difficult pill to swallow sometimes, but it works, and it has ensured stability since before humans learned to bash rocks together. Aimet is the master navigator of those systems. He's responsible for improving one of them for the first time in two thousand years."`
```